composer.json

```diff
@@ -20,16 +20,15 @@
     ],
     "license": "Apache-2.0",
     "require": {
-        "php": ">=7.1.0",
+        "php": ">=8.0",
         "illuminate/support": "~5.5|~6.0|~7.0|~8.0|~9.0|~10.0|~11.0",
         "illuminate/database": "~5.5|~6.0|~7.0|~8.0|~9.0|~10.0|~11.0",
         "illuminate/console": "~5.5|~6.0|~7.0|~8.0|~9.0|~10.0|~11.0",

```

Comment:
Please upgrade laravel to 9.0 or above. The following phpunit also needs to adjust the minimum version, and the same applies to other dependencies.